src/client/electron.ts

```diff
@@ -109,6 +108,13 @@ export class ElectronApplication extends ChannelOwner<channels.ElectronApplicati
     return result;
   }
 
+  async browserWindow(page: Page): Promise<JSHandle<BrowserWindow>> {
+    return this._wrapApiCall('electronApplication.browserWindow', async (channel: channels.ElectronApplicationChannel) => {
+      const result = await channel.browserWindow({ page: page._channel });
```

Comment:
I‘m interested that what will we get if `page` is actually an Electron `BrowserView`?